src/util/diproche/diprocheInterface.ts

```diff
@@ -0,0 +1,96 @@
+import IssueCode from "../../checking/issueCodes";
+import { getAllIssues } from "../../vocabularyChecker/detectWrongSyntax";
+import modes from "./predicateList.json";
+
+type Mode = "propositionalLogic" | "firstOrderPredicateLogic" | "test";
+
+export class UnexpectedError {
```

Comment:
```suggestion
export class UnexpectedError extends Error {
```

I think this is what @Khazuar meant this. This fixes the Error and should do what we want.

Reply:
You are right. Thanks!